### ISSUE TITLE
[K70 LUX RGB] After I've started it and used it. I close the program and need to restart my whole PC for it to work again.

### ISSUE THREAD

**Issue by bulletfever409** [antonpup/Aurora#381]
Please help.

**Comment by antonpup** [comment]
What do you mean? Did you exit Aurora completely? Or did you minimize it into the tray?

**Comment by bulletfever409** [comment]
I exit it completely as I only use it while on games that support it. But if I open it again after closing it the program won't pickup my keyboard. 

I'm using CUE2. And K70 LUX RGB

**Comment by bulletfever409** [comment]
The program also doesn't pickup my Corsair Harpoon RGB mouse

**Comment by antonpup** [comment]
Can you post logs? Should be in a folder called "logs" in the execution directory.

**Comment by bulletfever409** [comment]
http://pastebin.com/kfd5iNxa

**Comment by antonpup** [comment]
Looks like it's because of your mouse.
`Corsair device, Wrapper Exception! Message:Can't initial mouse with layout '0'`

Unplug it and see if keyboard works.

**Comment by bulletfever409** [comment]
Yep, that was the issue it seems. Any workaround you can think of?

**Comment by bulletfever409** [comment]
Anyway to force the program to stop trying to connect to my mouse?

**Comment by antonpup** [comment]
I can't think of anything. It's a problem with the SDK. Not Aurora itself.

**Comment by bulletfever409** [comment]
OK, so i've found a temporary work around. If you start the program up as admin then when the keyboard goes all black (lights go off) you unplug the mouse, wait for aurora to do it's thing and find the keyboard and then plug the mouse back in. It works perfectly, minus the mouse support.

TLDR:
1. start program while everything is plugged in and working.
2. unplug mouse when the keyboard lights are off.
3. wait for Aurora to pickup the keyboard (recommend having a desktop colour setup for this to be obvious.)
4. Plug the mouse back in and don't close(fully exit) aurora.

**Comment by antonpup** [comment]
I updated the CUE SDK on the dev build. But I am not sure if it will resolve your issue.

**Comment by bulletfever409** [comment]
How do I install the dev build? sorry, pretty noobish when it comes to github.

**Comment by simon-wh** [comment]
Try v0.6.2b, it should give a different result, closing for now
